tests/test_add.py

```diff
@@ -255,7 +255,24 @@ def test_ignore_name_ok(self, precached_repo: Repo):
 
         assert isolate_after.accessions == {"DQ178613", "DQ178614"}
 
-    def test_ignore_name_override_ok(self, precached_repo: Repo):
+    @pytest.mark.parametrize(
+        "expected_name, ignore_name, isolate_name",
+        [
+            (None, True, None),
+            (
+                IsolateName(type=IsolateNameType.ISOLATE, value="dummy"),
+                True,
+                IsolateName(type=IsolateNameType.ISOLATE, value="dummy")
+            ),
+            (
+                None,
+                True,
+                IsolateName(type=IsolateNameType.ISOLATE, value="dummy")
+            ),
+        ]
+    )
+    def test_ignore_name_override_ok(
+        self, precached_repo: Repo, expected_name: IsolateName | None, ignore_name: bool, isolate_name: IsolateName | None):
```

Comment:
Split parametrization into a class-based test with better docstrings. This will make it clear what is being tested after a few seconds of study.